szperacz/run.py

```diff
@@ -18,4 +18,23 @@
 
     files = discover_files(path)
 
-    print(files)
+    def found(files):
+        print(f'Images found: {len(files)}\n---')
+        for i in files:
+            print(f'{i}')
+
+        return '---'
+
+
+    def not_found(files):
+        return 'Images found: 0'
+
+
+    def founded(files):
+        if len(files) > 0:
+            return found(files)
+        else:
+            return not_found(files)
```

Comment:
1. Move definition of the functions outside main function body.

It does not suit any paradigm to have them such tightly coupled to the execution and may even cause some issues in future.

Further read: https://www.programiz.com/python-programming/closure

2. Please use more descriptive function names.

Those short, sharp names look good as methods in classes, but we are not OOP yet. It isn't clear what found/founded function does just from its name. 

How about: log_to_console (instead of founded), log_files (instead of found) and not_found can be removed and simple return statement could be used.